### DISCUSSION TITLE
Allow Default Repository for Project

### DISCUSSION THREAD

**Discussion by ericgroh** [community/community#11274] [Projects and Issues]
For Project Beta
Project or View should include a default repository setting for all items created in that project/view.

So when a new item is created, an issue for that repository is created.  Allowing the label project field and workflows to work without the extra step.

**Comment by etjernst** [comment]
Second this. Would be a useful option in settings.

**Comment by rodsmade** [comment]
There seems to be this option already, although I can't find it in my project's settings! https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/linking-a-repository-to-a-project-board
can you guys?

**Reply by ericgroh** [reply]
What you have linked is a feature of the old Projects.  From your profile page, if you create a project, non beta, those steps are accurate.

**Comment by vanlooverenkoen** [comment]
It would also be nice to have an option most/last used repository for that board at the top of the repository list.
I already created a ticket here -> https://github.com/github/feedback/discussions/11859
